Source/Networking/UploadManager.swift

```diff
@@ -50,14 +52,14 @@ struct UploadManager {
 	}
 
 	func upload(profile: TCCProfile, authMgr: NetworkAuthManager, siteInfo: (String, String)?, signingIdentity: SigningIdentity?, completionHandler: @escaping (Error?) -> Void) {
-		os_log("Uploading profile: %{public}s", profile.displayName)
+        logger.info("Uploading profile: \(profile.displayName)")
```

Comment:
Notice this was already using `os_log` and was declaring that the display name was public.  To make the `profile.displayName` always show up in the logs, this must include `, privacy: .public` within the string interpolation as well.

```suggestion
        logger.info("Uploading profile: \(profile.displayName, privacy: .public)")
```

Read more about it at the end of this [page of docs](https://developer.apple.com/documentation/os/osloginterpolation).